formwidgets/actionbuilder/assets/js/actions.js

```diff
@@ -60,15 +60,17 @@ function showActionSettings(id) {
     RuleActions.prototype.onShowNewActionSettings = function(actionId) {
         var $el = $('[data-action-id='+actionId+']')
 
+        var popup_size = 'giant';
```

Comment:
```suggestion
        var popup_size = 'giant'
```
Coding convention for JS in October is no semicolons.